doc/tutorials/11-ferrofluid/11-ferrofluid_part1.ipynb

```diff
@@ -482,8 +482,8 @@
    "source": [
     "## Exercise:\n",
     "\n",
-    "Remove overlap between particles by means of the steepest descent method. Run the steepest descent algorithm until the total energy of the system $E_{\\text{tot}} \\leq 5 k_{\\text{B}} T N$.\n",
-    "(Check the current energy every 20 minimization steps.)"
+    "Perform a steepest descent energy minimization.\n",
+    "Track the relative energy change $E_{\\text{rel}}$ per minimization loop (where the integrator is run for 10 steps) and terminate once $E_{\\text{rel}} \\le 0.05$."
```

Comment:
we should probably say that this corresponds to a 5 percent relative energy difference between two loops.